views/about.tmpl

```diff
@@ -59,18 +59,15 @@
     <div class="row">
       <div class="col-md-6 col-lg-4">
         <h2>On Air</h2>
-          <h3>AM Radio</h3>
-          <p>We are of course a legal radio station and we broadcast on AM radio all across campus on 1350KHz Medium Wave.</p>
-          <p>So what are you waiting for? Ask your parents if you can have their old boom box, and get it tuned into URY like it’s 1989.</p>
           <h3>FM Radio</h3>
           <p>We're also broadcasting across campus on 88.3MHz FM.</p>
-          <p>Did your parents get rid of the boom box at a car boot sale? Pick us up on that radio you got yourself because you're at uni and you're an "adult" now.</p>
+          <p>Do you own a radio or a car that contains a radio? Pick us up on that radio you got yourself because you're at uni and you're an "adult" now.</p>
```

Comment:
Slightly clumsy wording - what do you think about this:
```suggestion
          <p>Does your car have a radio? Did you buy yourself one because you're at uni and you're an "adult" now? Tune it to 88.3!</p>
```